rust-runtime/aws-smithy-http-server/src/plugin/alb_health_check.rs

```diff
@@ -43,7 +45,24 @@ pub struct AlbHealthCheckLayer<'a, HealthCheckHandler> {
 
 impl<'a> AlbHealthCheckLayer<'a, ()> {
     /// Handle health check requests at `health_check_uri` with the specified handler.
-    pub fn new<HandlerFuture: Future<Output = Response<BoxBody>>, H: Fn(Request<Body>) -> HandlerFuture>(
+    pub fn new_fn<HandlerFuture: Future<Output = StatusCode>, H: Fn(Request<Body>) -> HandlerFuture + Clone>(
```

Comment:
`from_handler` and `from_service` like [Operation](https://docs.rs/aws-smithy-http-server/0.55.1/aws_smithy_http_server/operation/struct.Operation.html)?

Reply:
Done in a961ecb40.